### PR TITLE
Adventure

### PR DESCRIPTION
**Pull request by amaliahern**
Starting with RSpec, FizzBuzz kata

### CODE REVIEW

File: 2_adventures/001/amaliahern/04/fizz_buzz.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def fizzbuzz(number)
+  return "fizzbuzz" if(number % 3 == 0 and number % 5 == 0)
```
Comment (kotp) [inline 2_adventures/001/amaliahern/04/fizz_buzz.rb:2]: Something to think about... if you chose not to return (yet) you can build your result and return when you are done.  This way you only have to check for each/all states one time.

Consider the case where you end up playing this game with 3 primes or 4 or 5?  How long will this one line get?

**Comment by kotp** [comment]
LGTM for day 4 submission.  We are on day 5 though...

**Comment by patmaddox** [comment]
Go you!

File: 2_adventures/001/amaliahern/04/fizz_buzz.rb (hunk 1 above)
Comment (amaliahern) [inline 2_adventures/001/amaliahern/04/fizz_buzz.rb:2]: Wow! Thank you for your answer. Endeed I am going to work on this now. You cool!! :-D

**Pull request merged**